x-pack/test/functional/apps/infra/logs/log_entry_categories_tab.ts

```diff
@@ -61,11 +61,17 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       before(() => logsUi.cleanIndices());
 
       it('Shows no data page when indices do not exist', async () => {
-        await logsUi.logEntryCategoriesPage.navigateTo();
-
-        await retry.try(async () => {
-          expect(await logsUi.logEntryCategoriesPage.getNoDataScreen()).to.be.ok();
-        });
+        await retry.tryWithRetries(
+          "retry if indices haven't been refreshed yet",
+          async () => {
+            await logsUi.logEntryCategoriesPage.navigateTo();
+
+            await retry.try(async () => {
```

Comment:
Will the outer `tryWithRetries` outlive the inner `try` in terms of `timeout`?

Reply:
Good catch, thank you. The timeout needs to be higher, of course. 🤦